src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -6,7 +6,7 @@
   -->
   <PropertyGroup>
     <OpenCoverVersion>4.6.519</OpenCoverVersion>
-    <ReportGeneratorVersion>2.5.0</ReportGeneratorVersion>
+    <ReportGeneratorVersion>2.5.9</ReportGeneratorVersion>
```

Comment:
Just to be safe instead of deleting these it might be worth add a condition to set them to these values if they aren't already set.